openhtf/exe/test_state.py

```diff
@@ -50,24 +48,24 @@ class TestState(object):
     config: The config being used for this test.
     test: openhtf.Test instance describing the test to run.
   """
-  State = Enum(
+  Status = Enum(
```

Comment:
With the overall upgrade of http output to include more robust framework state as well as test state, we needed a representative of framework state with access to the right data. The test executor became that representative. So what you call test execution status is more like framework status, whereas  test status refers to the current test run only.